src/assets/css/responsive.scss

```diff
@@ -14,881 +14,897 @@ $paragraph-color2: #d6d6d6;
 $white-color: #ffffff;
 $black-color: #000000;

```

Comment:
Did you merge master into this branch? All these white changes were made in the last PR merge we did.